src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/resourcetakes/view/ResourceTakesStyles.kt

```diff
@@ -0,0 +1,96 @@
+package org.wycliffeassociates.otter.jvm.app.ui.resourcetakes.view
+
+import javafx.geometry.Pos
+import javafx.scene.layout.BorderStrokeStyle
+import javafx.scene.paint.Color
+import javafx.scene.text.FontPosture
+import javafx.stage.Screen
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import tornadofx.*
+
+typealias LinearU = Dimension<Dimension.LinearUnits>
+
+class ResourceTakesStyles : Stylesheet() {
+    companion object {
+        val main by cssclass()
+        val leftRegionContainer by cssclass()
+        val rightRegion by cssclass()
+        val dragTarget by cssclass()
+        val contentText by cssclass()
+        val newTakeRegion by cssclass()
+        val contentScrollPane by cssclass()
+        val takesList by cssclass()
+
+        val takeMaxWidth = 500.px
+        val takeMinHeight = 80.px
+    }
+
+    private fun margin(top: LinearU, right: LinearU, bottom: LinearU, left: LinearU) = mixin {
+        backgroundInsets += box(top, right, bottom, left)
+        borderInsets += box(top, right, bottom, left)
+    }
+
+    private val topMargin = 15.px
+    private val bottomMargin = 15.px
+    private val leftRegionLeftMargin = 80.px
+    private val leftRegionRightMargin = 100.px
+
+    init {
+        main {
+            prefWidth = Screen.getPrimary().visualBounds.width.px - 600.0
+            prefHeight = Screen.getPrimary().visualBounds.height.px - 200.0
```

Comment:
Seems like a bad idea, and as a placeholder it will likely be forgotten to be fixed?

Reply:
Yeah I don't think this shouldn't be here anymore, I'll remove it